### PR TITLE
Add DIA inputs

### PR DESCRIPTION
**Pull request by countvajhula**
This adds the capital and ideas inputs.

### CODE REVIEW

**Comment by countvajhula** [comment]
@apromessi @jairtrejo This is ready for review!

File: input/capital-anonymized.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+* The infrastructure that enables clients of the project to use the project's services.
+* A portal that allows clients and members of the public to see the accounting results and outstanding balances owed.
+* The programming language that the project is implemented in.
+* The infrastructure that runs tests and other code quality checks during development.
+* The platform publicly hosting the source code repository.
+* The version control system that records project development.
+* A code linting tool, which is used to detect simple errors during development.
+* A test coverage monitoring service, which tracks changes in code coverage over time during development.
+* A compatibility testing tool that tests the code on many different versions of the language during development.
```
Comment (jairtrejo) [inline input/capital-anonymized.md]: I don't know about this one, I don't think we are actually using it or that it has helped development in any way.

File: input/capital-anonymized.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+* The infrastructure that enables clients of the project to use the project's services.
+* A portal that allows clients and members of the public to see the accounting results and outstanding balances owed.
+* The programming language that the project is implemented in.
+* The infrastructure that runs tests and other code quality checks during development.
+* The platform publicly hosting the source code repository.
+* The version control system that records project development.
```
Comment (jairtrejo) [inline input/capital-anonymized.md:6]: I don't know that this materially contributed to the project. Including git would suggest including our text editors, the electricity company, our moms... 😝

**Comment by jairtrejo** [comment]
Looks great to me! There are a couple things that I feel we could exclude, but I don't feel strongly about them.

**Review by jairtrejo — APPROVED** [review]

File: input/capital-anonymized.md (hunk 1 above)
Comment (countvajhula) [reply input/capital-anonymized.md]: We do use tox-gh-actions, but I suppose we don't use tox directly. Will remove it.

File: input/capital-anonymized.md (hunk 2 above)
Comment (countvajhula) [reply input/capital-anonymized.md:6]: With our text editors and our moms we're individually indebted to them :) With electricity we've already paid for it. But with Git, it's the project that is indebted to it and no one has paid for it. Adding Git also didn't initially occur to me, but in a way, recognizing projects as valuable as Git that have zero market value is exactly the kind of thing that ABE is meant to address!

**Comment by countvajhula** [comment]
I also updated the cost analysis (crudely, as always). A lot of the financial appraisal stuff is an art anyway so I don't think spending 80% extra effort in getting very precise labor estimates would be necessary. I'll go ahead and merge this and send it out, but we can still make changes as part of DIA on any inputs.

**Pull request merged**